Vagrantfile

```diff
@@ -13,10 +13,12 @@ apt-get update -qq
 apt-get install -y -qq --no-install-recommends chrony
 EOS
 
```

Comment:
I would support that, except that the entire reason for the change is that the process was pretty unreliable using Virtualbox.